tests/onboarding/api_regression/onboarding_regression_test_collection.json

```diff
@@ -0,0 +1,35 @@
+{
+	"info": {
+		"_postman_id": "25e21fc2-4872-4710-85cb-3760cd8f642b",
+		"name": "Ariel Onboard TPET API",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "36968695",
+		"_collection_link": "https://aws-educate-tw.postman.co/workspace/TPET-API-Development-Workspace~b6915213-64a3-408d-b297-45f472fa0a6b/collection/36968695-25e21fc2-4872-4710-85cb-3760cd8f642b?action=share&source=collection_link&creator=36968695"
+	},
+	"item": [
+		{
+			"name": "Onboarding Self Introduction",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "http://127.0.0.1:5000/onboarding/ariel",
```

Comment:
Hi @arielyu950329,

The URL should use the deployed AWS API Gateway endpoint, **not** your local one (127.0.0.1:5000).
For example, it could be:
`https://abcdefg123456.execute-api.us-west-2.amazonaws.com/onboarding/ariel`

If you encounter any difficulties, feel free to discuss them in the Slack channel. We’ll be there to help you with the configuration.